src/applications/vaos/services/appointment/transformers.js

```diff
@@ -364,12 +343,7 @@ export function transformVAOSAppointment(appt) {
       purpose.serviceName === (coding?.[0]?.code || coding) ||
       purpose.commentShort === (coding?.[0]?.code || coding),
   )?.short;
-  const comments =
-    appointmentComments.length > 0 ? appointmentComments[0] : appt.reasonCode;
-  const reasonCodeText = getReasonCodeDS(appt, 'comments')
-    ? getReasonCodeDS(appt, 'comments')
-    : comments?.text;
-  const text = appt.reasonCode ? reasonCodeText : null;
+  const text = appt.reasonCode ? appt.additionalAppointmentDetails : null;
```

Comment:
should we rename this to patientComments or something a little more descriptive that text?

Reply:
Not quite sure I understand. Are you talking about renaming 'appt.additonalAppointmentDetails' to 'patientComments' on the backend? That makes sense to me. Then I can change the name here.

Reply:
I think @simiadebowale meant `text` => `patientComments`

Reply:
yes, sorry my comment wasn't clear enough

Reply:
@JunTaoLuo - in vets-api maybe `patientComments` is much clearer than `additionalAppointmentDetails` as well, what do you think?

Reply:
Yea that makes sense to me too. Just FYI that we need to delay this PR to next sprint but IMO getting it done right is better than getting it done quickly 😄

Reply:
yup, I agree with that.

Reply:
@vbahinwillit - so we can update this PR `additionalAppointmentDetails` to `patientComments` but hold off on merging it until the vets api field is update

Reply:
Thanks for the review!